parsers/IN.py

```diff
@@ -165,8 +165,14 @@ def fetch_consumption(
 
     total_consumption = 0
     for state in STATES_MAPPING[zone_key]:
-        r: Response = session.get(DEMAND_URL.format(state=state))
-        soup = BeautifulSoup(r.content, "html.parser")
+        # By default the request headers are set to accept gzip.
+        # If this header is set, the proxy will not decompress the content, therefore we set it to an empty string.
```

Comment:
TIL

This is probably why I had such problems working with the proxies in the past 😅

Reply:
Yes, that was actually a bit hard to debug if you don't have access to the proxy code 😅

Reply:
So I think there's an issue on the backend the update doesn't seem to have been propagated, I'll have a look